internal/controller/components/trustyai/trustyai_support.go

```diff
@@ -30,6 +30,7 @@ var (
 		"guardrails-sidecar-gateway-image":   "RELATED_IMAGE_ODH_TRUSTYAI_VLLM_ORCHESTRATOR_GATEWAY_IMAGE",
 		"guardrails-built-in-detector-image": "RELATED_IMAGE_ODH_BUILT_IN_DETECTOR_IMAGE",
 		"ragas-provider-image":               "RELATED_IMAGE_ODH_TRUSTYAI_RAGAS_LLS_PROVIDER_DSP_IMAGE",
+		"garak-provider-image":               "RELATED_IMAGE_ODH_TRUSTYAI_GARAK_PROVIDER_IMAGE",
```

Comment:
has this new image in place? i could not find it on rhoai 3.2 branch

Reply:
https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/stone-prod-p02.hjvn.p1/product/ReleasePlanAdmission/rhoai/rhoai-onprem-v3-2-components-prod.yaml#L207-L208

IIRC, this image onboarding was deferred and @moulalis was working on it. However, based on the Image name `odh-trustyai-garak-lls-provider-dsp-rhel9`, The variable name should be `RELATED_IMAGE_ODH_TRUSTYAI_GARAK_LLS_PROVIDER_DSP_IMAGE`.


Also, @moulalis It will be nice if we can keep the component and the image name aligned. Component name should be `odh-trustyai-garak-lls-provider-dsp-v3-2`